raiden/constants.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from ethereum.utils import denoms, int_to_big_endian
```

Comment:
Maybe rename this to `options` or `settings`? As they are variables we decide on and not constants on the mathematical sense it makes a bit more sense to me.

Still not a hard requirement. Can stay as is.